AppCreationScripts/sample.json

```diff
@@ -0,0 +1,44 @@
+{
+    "Sample": {
+      "Title": "A Python Django Webapp to sign in users in your Azure AD tenant with the Microsoft Identity platform",
+      "Level": 100,
+      "Client": "Python Django Webapp",
+      "Service": "N/A",
+      "RepositoryUrl": "ms-identity-python-django-webapp-call-graph",
+      "Endpoint": "AAD v2.0"
+    },
+    "AADApps": [
+      {
+        "Id": "webApp",
+        "Name": "python-django-webapp-call-graph",
+        "Kind": "WebApp",
+        "Audience": "AzureADMyOrg",
+        "PasswordCredentials": "Auto",
+        "AllowImplicitFlow": false,
+        "HomePage": "http://localhost:8000",
+        "ReplyUrls": "http://localhost:8000/auth/redirect",
+        "RequiredResourcesAccess": []
+      }
+    ],
+    "CodeConfiguration": [
+      {
+        "App": "webApp",
+        "SettingKind": "Replace",
+        "SettingFile": "\\..\\aad.config.json",
```

Comment:
Scripts register a new app fine, but it doesn't update the `aad.config.json`. 

The path to aad.config.json ("SettingFile": "\\..\\aad.config.json",) looks correct though :/

could it something to do with virtual environment?